src/cross.ts

```diff
@@ -20,6 +25,8 @@ export type AccountData = {
   name: string;
   index: number;
   balance: string;
+  // chain-specific resources
+  hederaResources?: HederaResourcesRaw;
```

Comment:
@jackthta According to my understanding, the `accountId` could be retrieved on live-mobile side, in the same way it is retrieved on live-desktop, from the public key, which should be already provided in the AccountData. If you can please confirm, this would be a way to remove the need for `hederaResources` here.